.github/workflows/cut-release.yml

```diff
@@ -0,0 +1,59 @@
+name: "cut-release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version'
+        required: false
+jobs:
+  cut_release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code 
+        uses: actions/checkout@v3
+      - name: "Build & test"
+        run: |
+          sudo apt-get install build-essential -y
+          ./autogen.sh
+          ./configure
+          make distcheck
+      - name: Update VERSION file on checkout 
+        run: |
+          set -x
+          sudo apt-get install -y python3-semver
+          if [ -z "$NEW_VERSION" ]; then
+             echo "Current Version: $(cat VERSION)"
+             pysemver bump patch $(cat VERSION) > VERSION
```

Comment:
How about `NEW_VERSION=$(pysemver bump patch "$VERSION")` and then you can use common logic using `$NEW_VERSION`.

You might want to store `VERSION=$(cat VERSION)` at the start to be able to do comparisons, etc.